src/test/java/com/microsoft/graph/content/MSBatchRequestContentTest.java

```diff
@@ -85,4 +104,109 @@ public void testRemoveBatchRequesStepWithIdByAddingMultipleBatchSteps() {
         assertTrue(content.compareTo(expectedContent) == 0);
     }
 
+    @Test
+    public void defensiveProgrammingTests() {
```

Comment:
consider using JUnit constructs for expected exceptions: https://www.codingeek.com/tutorials/junit/assert-exception-thrown-junit/

Reply:
🤦‍♂️ I don't know how I didn't think of that earlier... Thanks a lot! makes the testing code much more readable!